package.json

```diff
@@ -63,7 +63,7 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "24.1.3",
     "eslint-plugin-prettier": "^3.3.1",
-    "husky": "^5.0.9",
+    "husky": "^4.3.8",
```

Comment:
“command not found” when using v5.

https://github.com/typicode/husky/issues/854